<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>imageInformationDialog</class>
 <widget class="QDialog" name="imageInformationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="lay_v_info">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="lay_h_setting">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QCheckBox" name="cbx_fil_pub">
         <property name="text">
          <string>公開設定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbx_fil_edit">
         <property name="text">
          <string>削除可能</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_fil_sta">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>ステータス：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_fil_stts">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_h_alias">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lab_fil_ali">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>別　　名：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbx_fil_ali"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_h_date">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lab_fil_dt_cr">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>作成日時：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dte_fil_dt_cre">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:zz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_fil_dt_cre_exif">
         <property name="text">
          <string>Exifから取得</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_fil_dt_mod">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>編集日時：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dte_fil_dt_mod">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="displayFormat">
          <string>yyyy-MM-dd HH:mm:zz</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_fil_dt_mod_exif">
         <property name="text">
          <string>Exifから取得</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_h_operation">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lab_fil_ope">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>操　　作：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmb_fil_eope">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lab_fil_ope_app">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>操作アプリ：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbx_fil_ope_app"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_h_caption">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lab_fil_cap">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>キャプション：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbx_fil_capt"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="lay_h_description">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="lbl_fil_dsc">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>備　　考：</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tbx_fil_dsc"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tab_src">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="laly_h_update">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="btn_fil_update">
       <property name="text">
        <string>更新する</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="box_fil_ope">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>box_fil_ope</sender>
   <signal>accepted()</signal>
   <receiver>imageInformationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>box_fil_ope</sender>
   <signal>rejected()</signal>
   <receiver>imageInformationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
